pyproject.toml

```diff
@@ -1,13 +1,43 @@
-# This is a template `pyproject.toml` file for snaps
-# This file is managed by bootstack-charms-spec and should not be modified
-# within individual snap repos. https://launchpad.net/bootstack-charms-spec
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
 
 [tool.flake8]
-ignore = ["C901", "D100", "D101", "D102", "D103", "W503", "W504"]
-exclude = ['.eggs', '.git', '.tox', '.venv', '.build', 'build', 'report']
 max-line-length = 99
+max-doc-length = 99
 max-complexity = 10
-classmethod-decorators = ["classmethod", "validator"]
```

Comment:
This configuration will be removed. May I ask what does it mean? I cannot find related document.

Reply:
ooh that's probably why flake8 is suggesting to add the `@classmethod` decorator to the methods above that have the `@validator` decorator.

This config option is to inform flake8 of any custom functions that are used to transform a method into a classmethod.

Reply:
I've seen other uses of the `@validator` decorator used with the `@classmethod` decorator, so I think it's fine to remove here and explicitly add the classmethod decorator.